Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,115 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponenet_GravityMovesEntity)
+    {
+        PhysX::CharacterControllerComponent* characterComponent = nullptr;
+        PhysX::CharacterGameplayComponent* gameplayComponent = nullptr;
+        AzFramework::TransformComponent* transform = nullptr;
+
+        float expected_gravity = 1.5f;
+
+        // Create character
+        auto gameplayEntity = AZStd::make_unique<AZ::Entity>("GameplayEntity");
+        {
+            auto characterConfiguration = AZStd::make_unique<Physics::CharacterConfiguration>();
+            auto characterShapeConfiguration = AZStd::make_unique<Physics::CapsuleShapeConfiguration>();
+            auto characterGameplayConfiguration = AZStd::make_unique<PhysX::CharacterGameplayConfiguration>();
```

Comment:
update: These don't need to be pointers, they can all just be created on the stack

```
auto characterConfiguration = Physics::CharacterConfiguration();
auto characterShapeConfiguration = Physics::CapsuleShapeConfiguration();
auto characterGameplayConfiguration = PhysX::CharacterGameplayConfiguration();
```

Reply:
Related to above comments/questions.

Reply:
If you want to have a quick chat about this it might be quicker to resolve, let me know if my comments above have cleared things up. Thanks!

Reply:
Yeah good plan, having a follow-up task to update these to the same style (removing the pointers) would be great. Thanks!